src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        src="./images/imac.jpeg" 
+        alt="mac-logo" 
+        class="card__logo"
+      >
+      <h1 class="card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <p class="card__secondtext">Product code: 195434</p>
+   
+
+      <div class="card__stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__reviews">
+          <p class="card__reviews-text">Reviews: 5</p>
+        </div>
+      </div>
+    
+      <div class="card__price">
+        <p class="card__price price">Price:</p>
+        <p class="card__price value">$2,199</p>
```

Comment:
why do you need price and value classes? use card__price-text and card__price-value

Reply:
because now it looks like separate blocks